utils/formValidations.ts

```diff
@@ -221,7 +215,7 @@ export function validateUserSubmittedFirstName(name: string): boolean {
 
 export function validateGoogleMapsUrlInput(url: string): boolean {
     url = url.trim()
-    if (url.startsWith('https://') && isValidUrl(url)) {
+    if (url.startsWith('https://') && url.includes('google.com/maps')) {
```

Comment:
```suggestion
    if (url.startsWith('https://') && url.includes('google.com/maps')) {
```

https://www.google.com/maps/